utils/pattern/src/parser/mod.rs

```diff
@@ -43,30 +43,36 @@ macro_rules! handle_literal {
 /// Options passed to the constructor of [`Parser`].
 ///
 /// ✨ *Enabled with the `alloc` Cargo feature.*
-#[derive(Debug)]
+#[derive(Debug, Default)]
 #[non_exhaustive]
 pub struct ParserOptions {
-    /// Controls whether ASCII letters can appear in the raw
-    /// pattern.
-    ///
-    /// If set to `true`, ASCII letters can be used directly in the pattern,
-    /// like "{0} days".
-    ///
-    /// If set to `false`, ASCII letters can only appear in quoted literals,
```

Comment:
does this actually mean ASCII letters? non-ASCII don't have to be quoted?

Reply:
This comes from datetime patterns, in which all ASCII letters are syntax characters.